### ISSUE TITLE
Update to work with kettle@2.

### ISSUE THREAD

**Issue by the-t-in-rtf** [fluid-project/fluid-express#49]
If run with kettle@2.0.0, the tests fail with:

```
14:49:39.476:  Sending a GET request to: http://127.0.0.1:6533/http://localhost:6533/hcf
14:49:39.482:  FATAL ERROR: Uncaught exception: Unexpected token T in JSON at position 0
SyntaxError: Unexpected token T in JSON at position 0
    at JSON.parse (<anonymous>)
    at fluid.test.express.helpers.verifyJSONContent (/Users/duhrer/Source/rtf/fluid-express/tests/js/lib/test-helpers.js:43:24)
    at togo (/Users/duhrer/Source/rtf/fluid-express/node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js:232:33)
    at wrapped (/Users/duhrer/Source/rtf/fluid-express/node_modules/infusion/tests/test-core/utils/js/IoCTestUtils.js:350:26)
    at fire (/Users/duhrer/Source/rtf/fluid-express/node_modules/infusion/src/framework/core/js/Fluid.js:1732:40)
    at IncomingMessage.<anonymous> (/Users/duhrer/Source/rtf/fluid-express/node_modules/kettle/lib/test/KettleTestUtils.http.js:162:13)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1244:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
```

**Comment by the-t-in-rtf** [comment]
The request payload was previously JSON as a string.  Now it's a string.

The [notes for the 2.0.0 release claim backward compatibility](https://github.com/fluid-project/kettle#notes-on-kettle-2x-releases), but something seems to have changed.

cc: @amb26

**Comment by the-t-in-rtf** [comment]
As discussed with Antranig, the errors were down to a change in the handling of the `path` and `url` fields that was addressed by the 2.1.0 release of kettle.
